scripts/templates.py

```diff
@@ -20,11 +22,12 @@ def read_yaml_files(yaml_dirs):
     data = {}
     for directory in yaml_dirs:
         for yaml_file in Path(directory).glob("config.yaml"):
-            with open(yaml_file, "r") as f:
-                try:
-                    data.update(yaml.safe_load(f))
-                except:
-                    ...
+            try:
+                config_data = yaml.safe_load(yaml_file.read_text())
+            except yaml.YAMLError:
+                print(f"Error reading {yaml_file}")
+                continue
+            data.update(unpack_config(config_data))
```

Comment:
We should probably evaluate how we want to handle yaml going forward so we don't end up with several. load_yaml() loads a yaml file using @dataclass to enforce schema on transformations yaml files. It makes it easier to do client-side validation, but going forward, the less logic we have client-side, the better (if we can get a config CDF service running).

Reply:
We need to consider to use `pydantic` for client side validation.